services/holidays/index.js

```diff
@@ -0,0 +1,125 @@
+/**
+ * Tabela da lua cheia de Páscoa, valida entre 1900 e 2199, inclusive
+ */
+function getPascalFullMoonDates() {
+  return [
+    [3, 14],
+    [3, 3],
+    [2, 23],
+    [3, 11],
+    [2, 31],
+    [3, 18],
+    [3, 8],
+    [2, 28],
+    [3, 16],
+    [3, 5],
+    [2, 25],
+    [3, 13],
+    [3, 2],
+    [2, 22],
+    [3, 10],
+    [2, 30],
+    [3, 17],
+    [3, 7],
+    [2, 27],
+  ];
+}
+
+/**
+ * Formata a data para meia noite, houveram alguns problemas com fuso
+ * horário e toLocaleString() causando a exibição da data anterior.
+ *
+ * @param Date date
+ * @return string
+ */
+function formatDateToMidnight(date) {
+  const formatedDay = `0${date.getDate()}`.substr(-2);
+  const formatedMonth = `0${date.getMonth() + 1}`.substr(-2);
+  return `${date.getFullYear()}-${formatedMonth}-${formatedDay} 00:00:00`;
```

Comment:
Dúvida: dado que o feriado ocorre no dia todo não seria melhor especificar apenas o dia sem horas? 😬 
`YYYY-MM-DD`

```suggestion
  return `${date.getFullYear()}-${formatedMonth}-${formatedDay}`;
```

Reply:
Obrigado pela revisão Luciano.
O motivo de eu ter feito com a hora zerada foi o _parse_ do `new Date()`, mas repensando agora não faz sentido resolver um problema do lado do cliente na API, acho que a sugestão faz todo o sentido, mas como afeta os testes e outros pontos precisa de uma atualização um pouco maior do que a sugerida.
A noite eu faço as alterações.